test/e2e/sample_e2e_test.go

```diff
@@ -114,12 +114,11 @@ var _ = Describe("Sample", func() {
 
 			// TODO: do we need to create catalog images are each scenario ? can we move building images to once per test suite (group of tests that uses the catalog)
 			// don't want to rebuild the image if it already exists
-			operatorImage := "quay.io/cdjohnson/ibm-sample-panamax-operator@sha256:47c9fcef261f3f26dbb6c05f904d11563aae43f61d72e0e4556399f3003c97d0"
-			operatorCommand := []string{"ibm-sample-panamax-operator"}
+			operatorImage := *dummyImage
```

Comment:
So I switched to use dummyImage is which is currently used by other e2e tests

Reply:
works for me

Reply:
not sure this is right... the default dummy image `bitnami/nginx:latest` is not an operator... its an operand.

Reply:
maybe its fine to use this as a fake operator... it just feels weird.